content/pt-br/docs/reference/access-authn-authz/abac.md

```diff
@@ -0,0 +1,137 @@
+       - `user`, tipo string; a string de usuário de `--token-auth-file`. Se você especificar `user`, ele deve corresponder ao nome do usuário autenticado.
+       - `group`, tipo string; se você especificar `group`, ele deve corresponder a um dos grupos do usuário autenticado. `system:authenticated` corresponde a todas as requi autenticadas. `system:unauthenticated` corresponde a todas as requisições não autenticadas.
+     - Propriedades de correspondência de recursos:
+       - `apiGroup`, tipo string; um grupo de API.
+         - Ex: `apps`, `networking.k8s.io`
+         - Curinga: `*` corresponde a todos os grupos de API.
+       - `namespace`, tipo string; um namespace.
+         - Ex: `kube-system`
+         - Curinga: `*` corresponde a todas as requisições de recursos.
+       - `resource`, tipo string; um tipo de recurso
+         - Ex: `pods`, `deployments`
+         - Curinga: `*` corresponde a todas as requisições de recursos.
+     - Propriedades sem correspondência de recursos:
+       - `nonResourcePath`, tipo string; caminhos de solicitação sem recurso.
+         - Ex: `/version` ou `/apis`
+         - Curinga:
+           - `*` corresponde a todas as requisições que não são de recursos.
+           - `/foo/*` corresponde a todos os subcaminhos de `/foo/`.
+     - `readonly`, tipo boolean, quando verdadeiro, significa que a política de correspondência de recursos se aplica apenas às operações `get`, `list` e `watch`, a política de não correspondência de recursos se aplica apenas à operação `get`.
+
+{{< note >}}
+Uma propriedade não definida é igual a uma propriedade definida com o valor zero para seu tipo
+(por exemplo, string vazia, 0, falso). No entanto, `unset` deve ser preferido para legibilidade.
+
+No futuro, as políticas poderão ser expressas no formato JSON e gerenciadas por meio de uma interface REST.
+{{< /note >}}
+
+## Algoritmo de Autorização
+
+Uma requisição possui atributos que correspondem às propriedades de um objeto de política.
+
+Quando uma requisição é recebida, os atributos são determinados. Atributos desconhecidos são definidos com o valor zero de seu tipo (por exemplo, string vazia, 0, falso).
+
+Uma propriedade definida como `"*"` corresponderá a qualquer valor do atributo correspondente.
+
+A tupla de atributos é verificada em relação a cada política do
+arquivo de política. Se pelo menos uma linha corresponder aos atributos da solicitação, a requisição é autorizada (mas pode falhar em validação posterior).
+
+Para permitir que qualquer usuário autenticado faça algo, escreva uma política com a propriedade do grupo definida como `"system:authenticated"`.
+
+Para permitir que qualquer usuário não autenticado faça algo, escreva uma política com a propriedade do grupo definida como `"system:unauthenticated"`.
+
+Para permitir que um usuário faça qualquer coisa, escreva uma política com as propriedades `apiGroup`, `namespace`, `resource`  e `nonResourcePath` definidas como `"*"`.
```

Comment:
Hi @stormqueen1990  - sorry, can you elaborate more on why that's not matching ? I've read it again and cannot get what's missing. 

First paragraph hi-lighted in the review covers unauthenticated users and second. 

Second one, the wildcard to allow a user to do anything, I just re-ordered and marked the properties as keywords for easier reading purposes.

Reply:
Indeed. Apologies, I think the difference in formatting confused me here. Please disregard this comment and feel free to mark the conversation as resolved.

Reply:
No worries! Thanks for all the detailed reviews, they help a lot in making sure quality is on point. I leave so many aspects behind that sometimes I feel my reviews take longer because of the lack of adhering to the standards we agreed upon .